db/migrate/20231030075543_create_edition_lead_images.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEditionLeadImages < ActiveRecord::Migration[7.0]
+  def change
+    create_table :edition_lead_images do |t|
+      t.integer "edition_id", foreign_key: true
+      t.integer "image_id", foreign_key: true
+      t.index %w[edition_id], name: "index_lead_image_on_edition_id", unique: true
+      t.index %w[image_id], name: "index_lead_image_on_image_id"
```

Comment:
Do we definitely need this index? Is there a scenario where we'll need to look up a lead image from the image? Deletion, maybe, if we can't load the lead image from the edition side 🤔

Reply:
Yea agreed, creating a joins table does them by default, but you're right it feels unlikely it'll be needed. I'll 🪓 it